android/app/src/main/java/host/exp/exponent/MainActivity.java

```diff
@@ -25,20 +44,36 @@ public String developmentUrl() {
     return DetachBuildConstants.DEVELOPMENT_URL;
   }
 
-  @Override
-  public List<String> sdkVersions() {
-    return new ArrayList<>(Arrays.asList("31.0.0"));
-  }
-
   @Override
   public List<ReactPackage> reactPackages() {
     return ((MainApplication) getApplication()).getPackages();
   }
 
   @Override
   public List<Package> expoPackages() {
-    // Here you can add your own packages.
-    return super.expoPackages();
+    return Arrays.<Package>asList(
+        new CameraPackage(),
+        new ConstantsPackage(),
+        new SensorsPackage(),
+        new FileSystemPackage(),
+        new FaceDetectorPackage(),
+        new GLPackage(),
+        new GoogleSignInPackage(),
+        new PermissionsPackage(),
+        new SMSPackage(),
+        new PrintPackage(),
+        new ConstantsPackage(),
+        new MediaLibraryPackage(),
+        new SegmentPackage(),
+        new FontLoaderPackage(),
+        new LocationPackage(),
+        new ContactsPackage(),
+        new BarCodeScannerPackage(),
+        new AdMobPackage(),
+        new StripePackage(),
+        new LocalAuthenticationPackage(),
+        new LocalizationPackage()
```

Comment:
it would be nice not to have to maintain two separate lists here and in ExperiencePackagePicker, but I see why you did this. Let's just make sure to document this well for when people add new unimodules